src/components/shared/MarkerTooltipContents.js

```diff
@@ -11,6 +11,19 @@ import formatTimeLength from '../../utils/format-time-length';
 import type { TracingMarker } from '../../types/profile-derived';
 import type { MarkerPayload } from '../../types/markers';
 
+function markerDetail(key, label, value) {
```

Comment:
Personally, I'd prefer stuff in the tooltips to be short. How about "Minor GCs" instead of "# Minor GCs" or "Minor GC Count"? Same for "Slices".

Actually, is the minor GC count useful? I've never paid much attention to it.

For the zone count, I would prefer something like "Zones: 3/7" (from timings.zones_collected and timings.total_zones).

While I'm asking for ponies, can we have the units after the max pause ("ms") and the nonincremental reason (but only if it was nonincremental)?